### ISSUE TITLE
Free energy calculation for Protein-protein binding with Openmm

### ISSUE THREAD

**Issue by mahdiofhijaz** [openmm/openmm#4471]
Dear OpenMM team,

I am looking for a package/tutorial to do protein-protein binding free energies. I have found plenty of alchemical methods for protein-ligand but none for protein-protein. Any pointers/tutorials would be of great help.

Many thanks!

**Comment by peastman** [comment]
I don't have any experience with that.  Perhaps someone else can provide information?

**Comment by swails** [comment]
If you're trying to find the absolute PPI free energies, the reason you're not seeing any tutorials is because this is an insanely hard problem.  Because proteins are generally quite large, alchemical approaches where you disappear/appear a protein would take ages to converge.  I suspect you'd have a higher likelihood of success if you used some kind of biasing potential to drive the proteins apart (either Umbrella Sampling or non-equilibrium methods like steered MD coupled with Jarzynski's equation to recover free energy profiles along the reaction coordinate).  The challenge to this approach is finding the right bespoke collective variable to use in the biasing potential.  Here's an [example paper that investigates this using the SIRAH coarse-grained potential](https://pubs.acs.org/doi/10.1021/acs.jctc.7b00660). What makes this approach hard to generalize is the bespoke nature of the collective variable -- it either needs to be generated, or at least inspected/validated for each system independently (not to mention the cost required to exhaustively sample each window and the solvent box size needing to be large enough to fit both proteins in the same box separated from each other in all periodic images by enough space.

If you want relative PPI free energies between a protein and a single-residue mutation, you can use alchemical methods to do that (mutating the single residue using either thermodynamic integration or free energy perturbation).  [Here's an example study using this approach.](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00348).  The trick here is to reformulate the problem into one that *looks* like small molecule binding, then treat it as you would a protein-ligand binding system.  As mentioned before, unless you can reduce the problem size to a mutation on the scale of a typical small-molecule ligand, TI or FEP are unlikely to converge.

-------

tl;dr unlike small molecule binding, PPIs are far more complex, and there is no pre-canned workflow that will work for the majority of studies that I'm aware of.
